src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -33,17 +33,17 @@ interface OutputInterface
     /**
      * Writes a message to the output.
      *
-     * @param bool                             $newline Whether to add a newline
-     * @param self::VERBOSITY_*|self::OUTPUT_* $options A bitmask of options (one of the OUTPUT or VERBOSITY constants),
-     *                                                  0 is considered the same as self::OUTPUT_NORMAL | self::VERBOSITY_NORMAL
+     * @param bool                                          $newline Whether to add a newline
+     * @param int-mask-of<self::VERBOSITY_*|self::OUTPUT_*> $options A bitmask of options (one of the OUTPUT or VERBOSITY constants),
```

Comment:
IIUC psalm expects a comma separated list:

```
Union types are not allowed in value-of type in docblock 
```

:shrug:

Reply:
Indeed, that even was my first version 😄 . Thx.